server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -1694,11 +1679,27 @@ protected NamedWriteableRegistry writeableRegistry() {
                     emptySet()
                 );
                 final IndexNameExpressionResolver indexNameExpressionResolver = TestIndexNameExpressionResolver.newInstance();
+                bigArrays = new BigArrays(new PageCacheRecycler(settings), null, "test");
```

Comment:
Suggest using a `MockBigArrays` sometimes (`rarely()` is enough) on the offchance that it finds a leak.